test/blackbox-tests/test-cases/melange/copy-files.t/bar/dune

```diff
@@ -0,0 +1,4 @@
+(library
+ (name bar))
```

Comment:
just missing the mode, no?

Reply:
you are right 🤦 Fixed in https://github.com/ocaml/dune/pull/6492/commits/e14b9e2b5493c411dd28262409c136ca49f071f5.

kind of related, I had a PR to show errors in these cases, but in the end dropped it because I spent a lot of time trying to make it work with Dune laziness and was not able to find a way 😞  #6412